python/cudf/cudf/core/indexed_frame.py

```diff
@@ -1800,6 +1800,60 @@ def drop_duplicates(
             self._index.names if not ignore_index else None,
         )
 
+    @_cudf_nvtx_annotate
+    def duplicated(self, subset=None, keep="first"):
+        """
+        Return boolean Series denoting duplicate rows.
+
+        Considering certain columns is optional.
+
+        Parameters
+        ----------
+        subset : column label or sequence of labels, optional
+            Only consider certain columns for identifying duplicates, by
+            default use all of the columns.
+        keep : {'first', 'last', False}, default 'first'
+            Determines which duplicates (if any) to mark.
+
+            - ``first`` : Mark duplicates as ``True`` except for the first occurrence.
+            - ``last`` : Mark duplicates as ``True`` except for the last occurrence.
+            - False : Mark all duplicates as ``True``.
+        """  # noqa: E501
+        if subset is None:
+            subset = self._column_names
+        elif (
```

Comment:
We repeat this kind of logic a lot for subsets by name(s). Is there a helper? Should there be?

Reply:
Fixed it